glibc.yaml

```diff
@@ -499,10 +500,10 @@ subpackages:
     description: "Password hashing library included with glibc"
     dependencies:
       provider-priority: 10
-    pipeline:
-      - runs: |
-          mkdir -p "${{targets.subpkgdir}}"/lib
-          mv "${{targets.destdir}}"/lib/libcrypt.so* "${{targets.subpkgdir}}"/lib/
+      provides:
+        - so:libcrypt.so.1=1
```

Comment:
hmm, this is probably going to give the wrong behavior.  all you need to do is just withdraw the old libcrypt1 packages, or set provides-priority on xcrypt.